src/orders/components/OrderRefundFulfilledProducts/OrderRefundFulfilledProducts.tsx

```diff
@@ -165,35 +167,18 @@ const OrderRefundFulfilledProducts: React.FC<OrderRefundFulfilledProductsProps>
                   </TableCell>
                   <TableCell className={classes.colQuantity}>
                     {line?.quantity ? (
-                      <TextField
+                      <QuantityInput
                         disabled={disabled}
-                        type="number"
-                        inputProps={{
-                          className: classes.quantityInnerInput,
-                          "data-test-id": "quantityInput" + line?.id,
-                          max: (line?.quantity ?? 0).toString(),
-                          min: 0,
-                          style: { textAlign: "right" },
-                        }}
-                        fullWidth
-                        value={selectedLineQuantity?.value}
+                        className={classes.quantityInnerInputNoRemaining}
+                        data-test-id={"quantityInput" + line?.id}
+                        value={Number(selectedLineQuantity?.value)}
```

Comment:
issue: This could result in a NaNNaNNaNBatman 😄 if the value is null or undefined, we should probably add a fallback:

```suggestion
                        value={Number(selectedLineQuantity?.value || 0)}
```